cmd/utils/cmd.go

```diff
@@ -85,6 +89,24 @@ func StartNode(stack *node.Node) {
 	}()
 }
 
+func monitorFreeDiskSpace(sigc chan os.Signal, path string) {
+	for {
+		freeSpace, err := getFreeDiskSpace(path)
+		if err != nil {
+			log.Warn("Failed to get free disk space", "path", path, "err", err)
+			break
+		}
+		if freeSpace < freeDiskSpaceCritical {
+			log.Error("Low disk space. Gracefully shutting down Geth to prevent database corruption.", "available", freeSpace/1024/1024)
```

Comment:
You can use `common.StorageSize` for these, something like this:

```suggestion
			log.Error("Low disk space. Gracefully shutting down Geth to prevent database corruption.", "available", common.StorageSize(freeSpace))
```

Reply:
good idea

Reply:
fixed: 17e224a